lib_stusb_impl/usbd_impl.c

```diff
@@ -353,21 +353,27 @@ static uint8_t const HID_ReportDesc_fido[] = {
 
 #define ARRAY_U2LE(l) (l)&0xFF, (l)>>8
 
+#define CFG_HDR_LEN    (0x9)
+#define CFG_HIDGEN_LEN (0x9+0x9+0x7+0x7)
+#define CFG_IO_U2F_LEN (0x9+0x9+0x7+0x7)
+#define CFG_USB_CCID_LEN (0x9+0x36+0x7+0x7)
+#define CFG_WEBUSB_LEN (0x9+0x7+0x7)
+
 /* USB HID device Configuration Descriptor */
-static __ALIGN_BEGIN uint8_t const USBD_CfgDesc[] __ALIGN_END =
+static __ALIGN_BEGIN uint8_t const N_USBD_CfgDesc[] __ALIGN_END =

```

Comment:
Should be done only if `#ifdef HAVE_USB_CLASS_CCID` to avoid impact on other apps?

Reply:
Done!